tbx/work/models.py

```diff
@@ -54,11 +46,28 @@ class WorkPageAuthor(Orderable):
         FieldPanel("author"),
     ]
 
+    class Meta:
+        abstract = True
+
+
+class HistoricalWorkPageScreenshot(Orderable, AbstractPageScreenshot):
+    page = ParentalKey("work.HistoricalWorkPage", related_name="screenshots")
+
+
+class HistoricalWorkPageAuthor(Orderable, AbstractPageAuthor):
+    page = ParentalKey("work.HistoricalWorkPage", related_name="authors")
+
+
+class HistoricalWorkPage(SocialFields, Page):
```

Comment:
I think we will need to rename the template file before this is merged the FE code has now been merged so if you rebase you can do this.

Reply:
Thanks @helenb I have rebased. Fromhttps://github.com/torchbox/torchbox.com/pull/49/files it seems the template is still `"patterns/pages/work/work_detail.html"` so I've not updated it here

Reply:
I thought we ought to rename it for clarity and consistency with the model name

Reply:
Updated in 6e5c4ff

Reply:
looks good!